include/node.h

```diff
@@ -272,6 +287,12 @@ class Node : public NodeBase<Tdim> {
   unsigned dof_{std::numeric_limits<unsigned>::max()};
   //! Status
   bool status_{false};
+  //! Scalar properties
+  tsl::ordered_map<mpm::properties::Scalar, Eigen::Matrix<double, 1, Tnphases>>
+      scalar_properties_;
+  //! Vector properties
+  tsl::ordered_map<mpm::properties::Scalar, Eigen::Matrix<double, 1, Tnphases>>
```

Comment:
Why `mpm::properties::Scalar`? Are we gonna put `Velocity` in `mpm::properties::Scalar`? Where to specify and store the dimensional components?

Reply:
That's just a place holder for now